users/serializers.py

```diff
@@ -28,6 +28,24 @@ def get_token(cls: Any, user: Any) -> Any:
         return token
 
 
+class ProfileSerializer(serializers.ModelSerializer):
+    username: Any = serializers.CharField(
+        read_only=True, source="user.username"
+    )
+    bio = serializers.CharField(allow_blank=True, required=False)
+    image = serializers.ImageField(use_url=True, required=False)
+
+    class Meta:
+        model = Profile
+        fields = ("username", "bio", "image")
+
+    def update(self, instance: Any, validated_data: Any) -> Any:
+        instance.bio = validated_data.get("bio")
```

Comment:
should be  instance.bio = validated_data.get("bio",  instance.bio) so that in case bio returns none it equates it to the current instance value same as for instance.image